lib/plain/common-types.ts

```diff
@@ -226,6 +226,14 @@ export type PlainClientAPI = {
       rawData: ContentTypeProps
     ): Promise<ContentTypeProps>
     unpublish(params: OptionalDefaults<GetContentTypeParams>): Promise<ContentTypeProps>
+    create(
+      params: OptionalDefaults<GetSpaceEnvironmentParams>,
+      rawData: CreateContentTypeProps
+    ): Promise<ContentTypeProps>
+    createWithId(
+      params: OptionalDefaults<GetSpaceEnvironmentParams & { contentTypeId: string }>,
+      rawData: ContentTypeProps
```

Comment:
This now requires `sys.environment`, `description` and `displayField` to be set. Those should be optional.